install_ffmpeg.sh

```diff
@@ -75,47 +75,88 @@ if [ ! -e "$ROOT/x264" ]; then
   make install-lib-static
 fi
 
+if [[ $(uname) == "Linux" && $BUILD_TAGS == *"debug-video"* ]]; then
+  if [ ! -e "$ROOT/x265" ]; then
+    sudo apt-get install -y libnuma-dev cmake
+    git clone https://bitbucket.org/multicoreware/x265_git.git "$ROOT/x265"
+    cd "$ROOT/x265"
+    git checkout 17839cc0dc5a389e27810944ae2128a65ac39318
+    cd build/linux/
+    cmake -DCMAKE_INSTALL_PREFIX=$ROOT/compiled -G "Unix Makefiles" ../../source
+    make
+    make install
+  fi
+  # VP8/9 support
+  if [ ! -e "$ROOT/libvpx" ]; then
+    git clone https://chromium.googlesource.com/webm/libvpx.git "$ROOT/libvpx"
+    cd "$ROOT/libvpx"
+    git checkout ab35ee100a38347433af24df05a5e1578172a2ae
+    ./configure --prefix="$ROOT/compiled" --disable-examples --disable-unit-tests --enable-vp9-highbitdepth --enable-shared --as=nasm
+    make
+    make install
+  fi
+fi
+
 EXTRA_FFMPEG_FLAGS=""
 EXTRA_LDFLAGS=""
+# all flags which should present for production build, but should be replaced/removed for debug build
+DEV_FFMPEG_FLAGS="--disable-programs"
+FFMPEG_MAKE_EXTRA_ARGS=""
 
 if [ $(uname) == "Darwin" ]; then
   EXTRA_LDFLAGS="-framework CoreFoundation -framework Security"
 else
   # If we have clang, we can compile with CUDA support!
   if which clang > /dev/null; then
     echo "clang detected, building with GPU support"
-
-    EXTRA_FFMPEG_FLAGS="--enable-cuda --enable-cuda-llvm --enable-cuvid --enable-nvenc --enable-decoder=h264_cuvid --enable-filter=scale_cuda,signature_cuda --enable-encoder=h264_nvenc"
-
+    EXTRA_FFMPEG_FLAGS="--enable-cuda --enable-cuda-llvm --enable-cuvid --enable-nvenc --enable-decoder=h264_cuvid,hevc_cuvid,vp8_cuvid,vp9_cuvid --enable-filter=scale_cuda,signature_cuda,hwupload_cuda --enable-encoder=h264_nvenc,hevc_nvenc"
     if [[ $BUILD_TAGS == *"experimental"* ]]; then
+        if [ ! -e "$ROOT/compiled/lib/libtensorflow_framework.so" ]; then
+          LIBTENSORFLOW_VERSION=2.3.0 \
+          && curl -LO https://storage.googleapis.com/tensorflow/libtensorflow/libtensorflow-gpu-linux-x86_64-${LIBTENSORFLOW_VERSION}.tar.gz \
+          && tar -C $ROOT/compiled -xzf libtensorflow-gpu-linux-x86_64-${LIBTENSORFLOW_VERSION}.tar.gz \
```

Comment:
Why do we have two tar commands here with `-C $ROOT/compiled` and `-C /usr/local`? Note that [Dockerfile.release-linux](https://github.com/livepeer/go-livepeer/blob/3649803de21ef36e1fe814c51ca6b94bef4daed6/docker/Dockerfile.release-linux#L17), which creates the final image with the go-livepeer binary, assumes that libtensorflow is installed in `/usr/local`.

Reply:
The short answer is because CircleCI build fails (Ffmpeg's configure cannot find libtensorflow) without copying the lib to system location, as it was implemented previously. I couldn't reproduce this issue by running docker build locally and don't think there's much harm in installing libtensorflow both system-wide and into $ROOT/compiled/lib.

Reply:
Ok, switched to `/usr/local` only installation, we can postpone figuring out why `configure` can't pick it up from $ROOT on CircleCI until we need a cross-platform build with Tensorflow.